papergres.go

```diff
@@ -724,10 +742,10 @@ func (r *Result) setMeta(m meta) {
 	r.LastInsertId.ID = m.LastInsertId
 	r.RowsAffected.Count = m.RowsAffected
 	if m.LastInsertId == 0 {
-		r.LastInsertId.Err = errors.New("No LastInsertId returned")
+		r.LastInsertId.Err = errors.New("no LastInsertId returned")
```

Comment:
Why change the case?

Reply:
Error strings should always be lowercased in Go.
For reference: https://github.com/golang/go/wiki/CodeReviewComments#error-strings